apps/site/assets/ts/schedule/components/ScheduleDirection.tsx

```diff
@@ -105,14 +102,20 @@ const ScheduleDirection = ({
       routePattern => routePattern.id === initialSelectedRoutePatternId
     ) || routePatternsInCurrentDirection.slice(0, 1)[0];
 
+  // If the route is a rail type, any other routePatterns
+  // of the current typicality are a branch
+  const branchPatterns = !isABusRoute(route)
+    ? routePatternsInCurrentDirection.filter(
+        pattern => pattern.typicality === defaultRoutePattern.typicality
+      )
+    : [];
+
```

Comment:
Aha!  The CR also needs a special case, because only the defaultRoutePattern should go on the map